app/styles/_overview.less

```diff
@@ -455,16 +455,49 @@
       }
     }
     .mobile-clients {
-      height: 30px;
-      margin-bottom: 1em;
-      overflow: hidden;
-      white-space: nowrap;
-      text-overflow: ellipsis;
-      a {
-        padding-left: 1em;
-        vertical-align: top;
+      .client-title {
+        color: #757575;
+      }
+
+      .mobile-client {
+        height: 60px;
+        margin-bottom: 1em;
+        overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
+
+        .client-icon {
+         vertical-align: bottom;
+         line-height: 42px;
+        }
+
+        .client-details {
+          display: inline-block;
+
+          .client-type {
+            padding-left: 10px;
+            color: #757575;
+            font-size: 11px;
+            line-height: 11px;
+          }
+        }
+
+        a {
+          padding-left: 1em;
```

Comment:
@sedroche i know this is very nit picky, but would it be possible to change this to `10px` as well, just to left align the 2 items?

Reply:
![image](https://user-images.githubusercontent.com/8274693/36722117-9fbb2e04-1ba4-11e8-8d29-fdc498f29b78.png)

Reply:
:+1: